From a0dc79a55b91ed15ca487b8c19d1877afce4c45b Mon Sep 17 00:00:00 2001
From: bbhtt <bbhtt.zn0i8@slmail.me>
Date: Sat, 7 Jun 2025 13:18:58 +0530
Subject: [PATCH] oci-builder: Add shadow to oci-builder compose

Users of initial_script in various `oci/` stacks may want to run
`useradd, userdel` etc. which is provided by shadow and the root
environment in which initial script commands are executed comes from
oci-builder.
---
 elements/oci/oci-builder.bst | 1 +
 1 file changed, 1 insertion(+)

diff --git a/elements/oci/oci-builder.bst b/elements/oci/oci-builder.bst
index b6b88871558..b54fc2695df 100644
--- a/elements/oci/oci-builder.bst
+++ b/elements/oci/oci-builder.bst
@@ -1,6 +1,7 @@
 kind: compose
 
 build-depends:
+- components/shadow.bst
 - oci/oci-builder-base.bst
 
 config:
-- 
GitLab

